test/sign_windows_test.py

```diff
@@ -0,0 +1,60 @@
+import io
+from contextlib import redirect_stdout
+from datetime import datetime, timedelta
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+from viewerutil.sign_windows import SignError, sign
+
+EXPIRES_FORMAT = '%b %d %H:%M:%S %Y'
+
+
+class SignWindowsTest(TestCase):
+    @patch('viewerutil.sign_windows.subprocess.run')
+    def test_fail(self, run):
+        run_return = MagicMock()
+        run_return.returncode = 1
+        run.return_value = run_return
+        with self.assertRaises(SignError) as ctx:
+            sign('executable-1',
+                 vault_uri='vault_uri',
+                 cert_name='cert_name',
+                 client_id='client_id',
+                 client_secret='client_secret',
+                 tenant_id='tenant_id',
+                 )
+        self.assertTrue('executable-1 signing failed' in str(ctx.exception))
```

Comment:
I like this test; thanks for the `unittest.mock.MagicMock` example.

The tests below exercise functionality we should remove: the new `AzureSignTool` doesn't report cert expiration the way its predecessor `signtool` does. Every run we get the "Failed to find certificate expiration date" warning.

Reply:
Alright, so should the certificate expiration date warning functionality be removed completely?